pages/badges.jsx

```diff
@@ -0,0 +1,192 @@
+                    </Link>
+                </div>
+            );
+        });
+
+        return (
+            <div>
+                <HeroUnit>
+                    <h1>Credentials</h1>
+                    <p className="text-center sub-title">Earn credentials to demonstrate you have the skills to teach the web.</p>
+
+                    <section>
+                        <IconLinks>
+                            <IconLink
+                                linkTo=""
+                                imgSrc="/img/pages/badges/svg/icon-explore.svg"
+                                head="Explore"
+                                subhead="See all badges we offer and which ones you qualify for."
+                                />
+                            <IconLink
+                                linkTo=""
+                                imgSrc="/img/pages/badges/svg/icon-earn.svg"
+                                head="Earn"
+                                subhead="Apply for badges by sharing your experiences."
+                                />
+                            <IconLink
+                                linkTo=""
+                                imgSrc="/img/pages/badges/svg/icon-share.svg"
+                                head="Share"
+                                subhead="Show your employers and friends you have the skills."
+                                />
+                        </IconLinks>
+                    </section>
+                </HeroUnit>
+
+                <div className="login-section">
+                    <div>
+                        <img src="/img/pages/badges/svg/divider.svg" width={'90%'} className="center-block horizontal-divider"/>
+                    </div>
+                    <div className={'text-center login-cta'}>
+                        <span className={'login-text'}>Sign in to start earning credentials.</span>
+                        <a href="/login" className="btn btn-awsm">Sign In</a>
+                    </div>
+                    <div>
+                        <img src="/img/pages/badges/svg/divider.svg" width={'90%'} alt="" className="center-block horizontal-divider"/>
+                    </div>
+                </div>
+
+
+
+                <div className="inner-container badges-content">
+                    <section>
+                        <h2 className="text-center">Earn 21st Century Skill Credentials</h2>
+                        <p className="text-center">Certain skills are critical to becoming a citizen of the web — like the ability to communicate, collaborate and create online. Hone these skills and earn badges that spotlight your expertise.</p>
+                        <div className="sep-16"></div>
+                        <div className="row">
+                            { badges }
+                        </div>
+                    </section>
+                </div>
+            </div>
+        );
+    }
+});
+
+module.exports = BadgesPage;
```

Comment:
Github likes to have newlines at the end of files. It's a bit silly, but easy to add in, so we tend to make sure files always end on a newline.